_data/sites.json

```
@@ -12373,6 +12411,17 @@
         ]
     },
 
+    {
+        "name": "Parallels",
+        "url": "https://my.parallels.com/profile/personal/general",
+        "difficulty": "hard",
```

We save hard for the ones that need customer support. Since this one seems to be fully accomplishable without involving another human, but takes a lot of steps, I think it can be changed to medium
```suggestion
        "difficulty": "medium",
```

---

I'll change that; forgot that medium existed :)